packages/lit-auth-client/src/lib/providers/BaseProvider.ts

```diff
@@ -112,6 +114,80 @@ export abstract class BaseProvider {
     return fetchRes.pkps;
   }
 
+  /**
+   * Fetch PKPs associated with given auth method type and id from pkp contract
+   *
+   * @param {AuthMethodType} authMethodType - Auth method type
+   * @param {string} authMethodId - Auth method id
+   *
+   * @returns {Promise<IRelayPKP[]>} - Array of PKPs
+   */
+  async getPKPsForAuthMethod({
+    authMethodType,
+    authMethodId,
+  }: {
+    authMethodType: AuthMethodType;
+    authMethodId: string;
+  }): Promise<IRelayPKP[]> {
+    if (!authMethodType || !authMethodId) {
+      throw new Error(
+        'Auth method type and id are required to fetch PKPs by auth method'
+      );
+    }
+
+    const litContracts = new LitContracts({
+      // Using a random wallet, we just need to read
+      signer: ethers.Wallet.createRandom(),
+    });
```

Comment:
the `LitContracts` instance is by default using the `cayenne` network, so we are actually fetching PKPs from the wrong intended network. 

```suggestion
    const litContracts = new LitContracts({
      randomPrivatekey: true,
      network: this.litNodeClient.config.litNetwork,
    });
```